tests/jobs_kafka_test.go

```diff
@@ -860,6 +860,93 @@ func TestKafkaPingOk(t *testing.T) {
 	assert.Equal(t, 1, oLogger.FilterMessage("ping kafka: ok").Len())
 }
 
+func TestKafkaTLS(t *testing.T) {
+	t.Run("PushJobTest", pushJobTest("configs/.rr-kafka-tls.yaml", "127.0.0.1:6002", "test-1-tls"))
+}
+
+func TestKafkaTLSRootCA(t *testing.T) {
+	t.Run("PushJobTest", pushJobTest("configs/.rr-kafka-tls-root-ca.yaml", "127.0.0.1:6003", "test-1-tls-root-ca"))
+}
+
+func pushJobTest(env string, rpcAddress string, pipeline string) func(t *testing.T) {
```

Comment:
Not bad 👍. But how did you know, what test caused an error, if they're both named `PushJobTest`? Furthermore, this is not a `PushJobTest`, these are tests with TLS and test with TLS and root CA.